west.yml

```diff
@@ -141,7 +141,7 @@ manifest:
     - name: nrfxlib
       repo-path: sdk-nrfxlib
       path: nrfxlib
-      revision: 741729d16c8ab0dd537d802d480180786951acad
+      revision: pull/1082/head
```

Comment:
Please update SHA: `293b78c621e70d24221f293a577a26e0bd621bf3`

Reply:
Or actually, please hold off, we have yet another PR (#12395) that moves nrfxlib SHA even further. We could update there and update 802154 repo revision exclusively here to avoid conflicts.